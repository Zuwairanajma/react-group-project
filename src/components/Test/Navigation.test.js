import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Navigations from '../Navigation';

test('Navigations component matches snapshot', () => {
  const { asFragment } = render(
    <MemoryRouter>
      <Navigations />
    </MemoryRouter>,
  );
  expect(asFragment()).toMatchSnapshot();
});
