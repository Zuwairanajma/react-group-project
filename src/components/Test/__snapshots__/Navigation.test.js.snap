// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigations component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="parent-header"
  >
    <nav
      class="Nav-start"
    >
      <img
        alt="Logo"
        class="Logo"
        src="spaceshiplogo.png"
        style="width: 60px;"
      />
      <h2>
        Space Travellers' Hub
      </h2>
    </nav>
    <ul
      class="lists-style"
      style="display: flex; justify-content: flex-end; gap: 20px; flex: 3;"
    >
      <li
        class="mappedlinks"
        style="list-style: none;"
      >
        <a
          aria-current="page"
          class="active-link"
          href="/"
        >
          Rockets
        </a>
      </li>
      <li
        class="mappedlinks"
        style="list-style: none;"
      >
        <a
          class="link"
          href="/Mission"
        >
          Mission
        </a>
      </li>
      <li
        class="mappedlinks"
        style="list-style: none;"
      >
        <a
          class="link"
          href="/MyProfile"
        >
          My Profile
        </a>
      </li>
    </ul>
  </div>
</DocumentFragment>
`;

exports[`Navigations component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="parent-header"
  >
    <nav
      class="Nav-start"
    >
      <img
        alt="Logo"
        class="Logo"
        src="spaceshiplogo.png"
        style="width: 60px;"
      />
      <h2>
        Space Travellers' Hub
      </h2>
    </nav>
    <ul
      class="lists-style"
      style="display: flex; justify-content: flex-end; gap: 20px; flex: 3;"
    >
      <li
        class="mappedlinks"
        style="list-style: none;"
      >
        <a
          aria-current="page"
          class="active-link"
          href="/"
        >
          Rockets
        </a>
      </li>
      <li
        class="mappedlinks"
        style="list-style: none;"
      >
        <a
          class="link"
          href="/Mission"
        >
          Mission
        </a>
      </li>
      <li
        class="mappedlinks"
        style="list-style: none;"
      >
        <a
          class="link"
          href="/MyProfile"
        >
          My Profile
        </a>
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
